packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -127,18 +128,6 @@ const getAppEntryPoint = () => {
     return resolve('./', EXT_OVERRIDES_DIR_NO_SLASH, 'app', 'main')
 }
 
-const getPublicPathEntryPoint = () => {
-    return resolve(
-        projectDir,
-        'node_modules',
-        '@salesforce',
-        'pwa-kit-dev',
-        'ssr',
-        'server',
-        'public-path'
-    )
-}
```

Comment:
This function was something I added back in the original base path PR from last year to support webpack hot reloading. Instead of defining the hmrBasePath in a different file, we now just define it in this file.